### ISSUE TITLE
[Bug] : the node_module folder is not availabel in .gitignore file

### ISSUE THREAD

**Issue by Rohit-Sharmaa** [SxxAq/TodoWebApp#17]
Description
The node_module folder is in .gitignore file it is not added in .gitignore file.

Additional Context
You never push the node_modules in the repository. Instead what you commit is package.json and package-lock.json files.

ScreenShot
<img width="566" alt="image" src="https://github.com/SxxAq/TodoWebApp/assets/130988755/0a0e3efc-3b3f-494c-a143-86b63e872866">

**Comment by SxxAq** [comment]
> Description The node_module folder is in .gitignore file it is not added in .gitignore file.
> 
> Additional Context You never push the node_modules in the repository. Instead what you commit is package.json and package-lock.json files.
> 
> ScreenShot <img alt="image" width="566" src="https://user-images.githubusercontent.com/130988755/270998811-0a0e3efc-3b3f-494c-a143-86b63e872866.png">

**Node_modules is not meant to be pushed to the repository, thats why it is listed in the .gitignore file.**

**Comment by Rohit-Sharmaa** [comment]
@SxxAq  I agree with you that node_modules file is not be pushed into github but it is good practice to stash the  node_module folder instead of add the node_module file in .gitignore file
